index.html

```diff
@@ -0,0 +1,178 @@
+                    <span class="material-symbols-outlined">
+                        gallery_thumbnail
+                    </span>
+                    <h3>CC Exhibition</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        forum
+                    </span>
+                    <h3>Forum</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        note_alt
+                    </span>
+                    <h3>Workshop</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        hub
+                    </span>
+                    <h3>CC Ignite</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+            </div>
+
+            <a class="see-program-desktop">SEE THE WHOLE PROGRAM</a>
+            <a class="primary-btn">Join Digital Marketing Summit 2023</a>
+        </div>
+    </section>
+    <section class="speakers">
+        <div class="container">
+            <h2>Featured Speakers</h2>
+            <hr class="one">
+            <div class="speaker-list hideContent"></div>
+            <div class="show-more">
+                <a href="#/">MORE</a><span class="material-symbols-outlined">
+                    expand_more
+                </span>
+            </div>
+        </div>
+    </section>
+    <section class="desktop-partners">
+        <div class="container">
+
+            <h2>Partner</h2>
+            <hr>
+            <div class="social-media-logos">
+                <h3>mozilla</h3>
+                <h3>Google</h3>
+                <h3>Naver</h3>
+                <h3>airbnb</h3>
+                <h3>Facebook</h3>
+
+            </div>
+
+        </div>
+    </section>
+    <footer class="desktop-footer">
+        <div class="container">
+
+            <h2>Digital Marketing Summit</h2>
+            <div class="rights">
+                <p>2023 Digital Marketing.</p>
+                <p>Some Rights Reserved.</p>
+            </div>
+        </div>
+
+    </footer>
```

Comment:
- [ ] Kindly note that the content of the footer is not yet well aligned, there has to be space and they should be at the center for the desktop version.

## Your design

![image](https://user-images.githubusercontent.com/86200356/233100580-398cb0c8-b1cf-436e-b103-1e2182e2ff5a.png)

## Behance design

![image](https://user-images.githubusercontent.com/86200356/233100489-b5ce5789-b259-4ff0-bca7-8a400413c519.png)